styles/icons/main.scss

```diff
@@ -12,7 +12,9 @@ $icon-size: 16px !default;
     src: embed-url('KendoUIGlyphs.woff') format('woff');
 }
 
-.k-icon {
+.k-icon,
+.k-grouping-dropclue
```

Comment:
This rule seems inappropriate for this file. The relevant styles seem to be [already copied in the grid](https://github.com/telerik/kendo-theme-default/pull/54/files#diff-39e608bd4112f0dbcad043205e94907eR225). Or perhaps it is a better idea to render the `k-icon` class on the element?